app-server/migrations/20240211183741-create-task.js

```diff
@@ -1,34 +1,34 @@
-'use strict';
+"use strict";
 /** @type {import('sequelize-cli').Migration} */
 module.exports = {
   async up(queryInterface, Sequelize) {
-    await queryInterface.createTable('tasks', {
+    await queryInterface.createTable("tasks", {
       id: {
         allowNull: false,
         autoIncrement: true,
         primaryKey: true,
-        type: Sequelize.INTEGER
+        type: Sequelize.INTEGER,
```

Comment:
Use UUID for the primary key, UUIDs are more unique than integers in nature. You need to install an extension on your database by running a migration which will do `CREATE EXTENSION IF NOT EXISTS "uuid-ossp"`. 

Then you can use `defaultValue: Sequelize.literal('uuid_generate_v4()'),` for your primary key and let postgres itself assign the UUIDs to your entities.